### ISSUE TITLE
Start a content repository

### ISSUE THREAD

**Issue by pglevy** [Bixal/rapid-response-team#6]
Some content types to consider:

- [x] offices
- [ ] bios
- [ ] blog posts
- [x] photos (see USDA and others on Flickr)
- [x] faq
- [ ] search results
- [ ] news
- [ ] podcasts and videos (embeds, for track list template)
- [ ] services
- [ ] account info (users)
- [ ] tweets

**Comment by pglevy** [comment]
For this first cycle, just want to get a few different examples going.

**Comment by bnaolu** [comment]
Super neat! These look great so far. I'm assuming this is content we'll be able to pull into our page templates?

**Comment by pglevy** [comment]
Yes, we can pull into page templates, and Figma files as well. Obviously still a lot to do here, but the main goal for this cycle was just to get this repo started in some kind of organized way. We'll continue to add to it and iterate on the presentation as we go. 

Just added in pages for the [images](https://bixal.github.io/rrt-content/images/) and [products](https://bixal.github.io/rrt-content/contraceptive-products/) that we used for templates and prototyping this cycle so they're accessible for reuse.

Closing for now.
